apps/hash-graph/lib/graph/src/store/postgres/query/compile.rs

```diff
@@ -283,31 +283,6 @@ impl<'c, 'p: 'c, R: PostgresRecord> SelectCompiler<'c, 'p, R> {
         }
     }
 
-    fn compile_latest_entity_version_filter(
```

Comment:
We use temporal versioning to determine "latest", it was forgotten to be removed.